appveyor.yml

```diff
@@ -39,9 +39,11 @@ before_build:
 
 build_script:
   - Rscript -e "install.packages('https://github.com/Open-Systems-Pharmacology/rClr/releases/download/v0.9.1/rClr_0.9.1.zip', repos = NULL, type = 'binary')"
+  - Rscript -e "install.packages('https://ci.appveyor.com/api/buildjobs/s4nhbp52qrknf40m/artifacts/tlf_1.2.43.zip', repos = NULL, type = 'binary')"

```

Comment:
if you want to automatically use the latest available version of tlf on AppVeyor, you could use the permalink like e.g. in the Reporting Engine:
https://github.com/Open-Systems-Pharmacology/OSPSuite.ReportingEngine/blob/dbd1435fec9bee4559dad8695484fd2d520457f7/appveyor.yml#L38